virl2_client/models/cl_pyats.py

```diff
@@ -120,6 +122,36 @@ def sync_testbed(self, username: str, password: str) -> None:
         self._testbed = self._load_pyats_testbed(testbed_yaml)
         self.set_termserv_credentials(username, password)
 
+    def switch_pyats_serial_console(self, node_label: str, console_number: int) -> None:
+        """
+        Switch to different serial console that is used to execute PyAts commands
+        should be executed after sync_testbed
+        and re-executed after every sync_testbed call.
+
+        :param node_label: The label/title of the device.
+        :param console_number: The serial console number to be used for PyAts.
+        """
+
+        try:
+            pyats_device: Device = self._testbed.devices[node_label]
+        except KeyError:
+            raise PyatsDeviceNotFound(node_label)
+
+        node: Node = [node for node in self._lab.nodes()if node.label == node_label].pop()
+
+        # will raise API error inside if console does not exist for device
+        node.console_key(console_number)
```

Comment:
you don't want to store and use the key?  what's the reason of calling it then?  I'd let it fail later